test/regression.uts

```diff
@@ -8034,10 +8039,10 @@ assert(r6 == "d279:1205:e445:5a9f:db28:efc9:afd7:f594")
 
 random.seed(0x2807)
 r6 = RandIP6("2001:db8::-") 
-assert(r6 == "2001:0db8::afd7")
+assert(r6 == "2001:0db8::e445")
```

Comment:
Those changes is beause the new `__cmp__` methods calls `_fix()` only once (the legacy one called it between 1-3 times)